keycloak/keycloak-admin-ui
# Adds selectable attribute functionality

## Motivation


Selectable attribute functionality is necessary for the Clients -> Evaluate screens. This PR allows the user to set a prop on `AttributeForm` called `isKeySelectable`, which would render a dropdown for the key of attribute pairs. 
Towards https://marvelapp.com/prototype/5eg9ic7/screen/83188830

## Verification Steps


1. Go to `RealmRoleTabs.tsx`.
2. Find the `AttributesForm` component on line 385.
3. Add the prop `isKeySelectable`
4. Add the prop `selectableItems={["a", "b", "c"]}`
5. Go to Realm Roles.
6. Create a new realm role, save it, then go to the "Attributes" tab.
4. Verify that for all attribute keys, you are seeing a Select dropdown with the values you entered.
5. Add/save some attributes and verify that the data is saved and shown correctly.
